csharp-advanced-topics/StaticClasses/StaticClasses/CollegeStudent.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace StaticClasses
+{
+    public class CollegeStudent
+    {
+        //private fields 
+        private string _name;
+
+        //non-static properties
+        public int Id { get; set; }
+        public string Name { get { return _name; } set => _name = value; }
+        public DateTime DateOfBirth { get; set; }
+
+        //non-static methods
+        //function to return the student's age
+        public int CalculateAge(DateTime DateOfBirth)
+        {
+            //get today's date 
+            var _today = DateTime.Today;
+
+            //calculate age
+            var _age =_today.Year - DateOfBirth.Year;
```

Comment:
Local variables shouldn't have underscores. Only the private fields should have those, like the _name. Please go through the entire solution and fix it where applicable.